### DISCUSSION TITLE
deprecated warning when using `shallow`

### DISCUSSION THREAD

**Discussion by moklick** [pmndrs/zustand#2133] [Bug report]
Hey!

Some users mentioned (wbkd/react-flow#3494), that they see a warning:

> [DEPRECATED] Default export is deprecated. Instead use `import { shallow } from 'zustand/shallow'`.

It seems, that that it is an issue when using Zustand with Parcel. As you can see in [this codesandbox](https://codesandbox.io/s/parcel-forked-496h8z?file=/src/index.js ), we see the warning, even if we are doing the import correctly:

```js 
import { shallow } from "zustand/shallow";

const result = shallow({}, {});
```

Is there anything that we can do about it? Or should we now use `useShallow` ?

**Comment by dbritto-dev** [comment]
@moklick there's a PR to fix that -> https://github.com/pmndrs/zustand/pull/2128

**Reply by moklick** [reply]
How could I miss that?! Thanks @dbritto-dev 🙏
